pulseguardian/web.py

```diff
@@ -414,7 +454,10 @@ def register_handler():
         return render_template('register.html', email=email,
                                signup_errors=errors)
 
-    PulseUser.new_user(username, password, g.user)
+    # Default owner to current user in case they emptied the field or had a
+    # typo that mangled the emails and none match.
```

Comment:
I think it might be better to reject this if the list is non-empty but can't be parsed.